lib/matplotlib/axes/_axes.py

```diff
@@ -6542,70 +6542,75 @@ def psd(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
 
         If len(*x*) < *NFFT*, it will be zero padded to *NFFT*.
 
-          *x*: 1-D array or sequence
+        Parameters
+        ----------
+        x : 1-D array or sequence
             Array or sequence containing the data
 
         %(Spectral)s
```

Comment:
Some common arguments in all the mlab functions. They're defined [here](https://github.com/matplotlib/matplotlib/pull/7359/files#diff-23e94f84624407f9051f33a8c4c79464R814).